system/services/formbuilder/FormBuilderService.cfc

```diff
@@ -1141,6 +1142,104 @@ component {
 		);
 	}
 
+	/**
+	 * Delete the given submission files.
+	 *
+	 * @autodoc
+	 * @submissionId.hint The ID of the submission
+	 *
+	 */
+	public void function deleteSubmissionFiles( required string submissionId ) {
+		var submission    = getSubmission( submissionId=arguments.submissionId );
```

Comment:
I wonder if there is anyway to background task this _after_ the form / response has been deleted. If we delete a large form with a lot of entries, this process could take a long while and create a very slow request.

Easiest way perhaps would be to make the delete form flow use an adhoc bg task with a progress UI for the user to view it. Same for submission. That way, the task can take a while and the user can navigate away, etc.